tests/integration/fixtures/mini_sentry.py

```diff
@@ -308,16 +308,24 @@ def get_project_config():
         if relay_id not in authenticated_relays:
             abort(403, "relay not registered")
 
+        response = {}
         configs = {}
         pending = []
+        global_ = None
+
         version = flask_request.args.get("version")
         if version in [None, "1"]:
             for project_id in flask_request.json["projects"]:
                 project_config = sentry.project_configs[int(project_id)]
                 if is_trusted(relay_id, project_config):
                     configs[project_id] = project_config
 
-        elif version in ["2", "3"]:
+        elif version in ["2", "3", "4"]:
+            if version == "4":
+                if flask_request.json["global"]:
```

Comment:
Even in v4, this will be an optional field. Note how the other fields are required, hence we should still fail if they are missing.

```suggestion
                if flask_request.json.get("global"):
```